tests/test_edgeql_ddl.py

```diff
@@ -1043,6 +1043,182 @@ async def test_edgeql_ddl_28(self):
             };
         """)
 
+    async def test_edgeql_ddl_drop_extending_01(self):
+        await self.con.execute("""
+            SET MODULE test;
+
+            CREATE TYPE Parent {
+                CREATE PROPERTY name -> str {
+                    CREATE CONSTRAINT exclusive;
+                };
+            };
+            CREATE TYPE Child EXTENDING Parent;
+        """)
+
+        await self.con.execute("""
+            ALTER TYPE Child DROP EXTENDING Parent;
+        """)
+
+        async with self.assertRaisesRegexTx(
+            edgedb.QueryError,
+            "object type 'test::Child' has no link or property 'name'",
+        ):
+            await self.con.execute("""
+                SELECT Child.name
+            """)
+
+        # Should be able to drop parent
+        await self.con.execute("""
+            DROP TYPE Parent;
+        """)
+
+    async def test_edgeql_ddl_drop_extending_02(self):
+        await self.con.execute("""
+            SET MODULE test;
+
+            CREATE TYPE Parent {
+                CREATE PROPERTY name -> str {
+                    CREATE CONSTRAINT exclusive;
+                };
+            };
+            CREATE TYPE Child EXTENDING Parent {
+                ALTER PROPERTY name {
+                    SET OWNED;
```

Comment:
Note: you don't need to `SET OWNED` if there are other operations in your `ALTER`, it's set implicitly since you're making local alters.

Reply:
Roger. In this case I think it makes things more clear.